lib/redix/pubsub/connection.ex

```diff
@@ -230,6 +259,19 @@ defmodule Redix.PubSub.Connection do
     end
   end
 
+  def connected({:call, from}, :ping, data) do
+    random_string = :crypto.strong_rand_bytes(12) |> Base.encode64()
+    pipeline = [["PING", "ping:#{random_string}"]]
+    data.transport.send(data.socket, Enum.map(pipeline, &Protocol.pack/1))
```

Comment:
Need to handle if this returns something that is not `:ok` here, and disconnect. We do it in other places too.